homeassistant/components/climate/maxcul.py

```diff
@@ -0,0 +1,269 @@
+"""
+Support for MAX! thermostats using the maxcul component.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/climate.maxcul/
+"""
+import logging
+import os
+
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util.yaml import load_yaml, dump as dump_yaml
+from homeassistant.components.climate import (
+    ClimateDevice, SUPPORT_TARGET_TEMPERATURE, SUPPORT_OPERATION_MODE,
+    PLATFORM_SCHEMA
+)
+from homeassistant.const import (
+    TEMP_CELSIUS, ATTR_TEMPERATURE
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['pymaxcul==0.1.8']
+
+DOMAIN = 'maxcul'
```

Comment:
Remove this, it's only for a platform

Reply:
I have a service on that domain, where should I put the service then?